addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -110,7 +110,12 @@
     <t t-set="body_classname" t-value="'o_wslides_body'"/>
     <t t-call="website.layout">
         <div id="wrap" t-attf-class="wrap mt-0">
-            <div t-attf-class="o_wslides_course_header o_wslides_gradient position-relative text-white pb-md-0 pt-2 pt-md-5 #{'pb-3' if channel.channel_type == 'training' else 'o_wslides_course_doc_header pb-5'}">
+            <t t-call="website.record_cover">
+            <t t-set="_record" t-value="channel"/>
```

Comment:
OK to not indent to keep the diff concise ? Happens in 2 places for about 40 lines difference for each of them.

Reply:
A bit annoying for diff but I would indent. Otherwise you are quickly lost, not know which are the parents of current html tag you are modifying. Anyway adding w=1 in git diff allows to reduce displayed diff when there is only indent diff.